contracts/oracles/sources/oracle.move

```diff
@@ -0,0 +1,38 @@
+module oracles::oracle {
```

Comment:
Write a small documentation for this module. You can look at other modules in such as [this](https://github.com/embedr-finance/embedr-sui-contracts/blob/phase-2/contracts/stable_coin_factory/sources/kasa_operations.move) or [this](https://github.com/embedr-finance/embedr-sui-contracts/blob/phase-2/contracts/stable_coin_factory/sources/stability_pool.move) for more info.